app/controllers/pensions/unauthorisedPayments/UnauthorisedPensionSchemeTaxReferenceController.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.pensions.unauthorisedPayments
+import config.{AppConfig, ErrorHandler}
+import controllers.predicates.AuthorisedAction
+import forms.PensionSchemeTaxReferenceForm
+import models.mongo.PensionsCYAModel
+import play.api.data.Form
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.PensionSessionService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import utils.Clock
+import views.html.pensions.unauthorisedPayments.PensionSchemeTaxReferenceView
+import controllers.pensions.routes.PensionsSummaryController
+import controllers.predicates.TaxYearAction.taxYearAction
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class UnauthorisedPensionSchemeTaxReferenceController @Inject()(implicit val cc: MessagesControllerComponents,
+                                                                authAction: AuthorisedAction,
+                                                                pensionSchemeTaxReferenceView: PensionSchemeTaxReferenceView,
+                                                                appConfig: AppConfig,
+                                                                pensionSessionService: PensionSessionService,
+                                                                errorHandler: ErrorHandler,
+                                                                clock: Clock,
+                                                                ec: ExecutionContext) extends FrontendController(cc) with I18nSupport {
+
+
+  def show(taxYear: Int): Action[AnyContent] = (authAction andThen taxYearAction(taxYear)).async{ implicit request =>
+    pensionSessionService.getPensionSessionData(taxYear, request.user).flatMap{
+      case Left(_) => Future.successful(errorHandler.handleError(INTERNAL_SERVER_ERROR))
+      case Right(Some(pensionsUserData)) => {
+        val emptyForm: Form[String] = PensionSchemeTaxReferenceForm.pensionSchemeTaxReferenceForm(request.user.isAgent)
```

Comment:
At the moment we don't have the capability to add or reference a particular pension scheme tax reference, however i think you should add a TODO here to remind the person implementing the next story that it needs to be updated here